### ISSUE TITLE
Will it support irb?

### ISSUE THREAD

**Issue by ifyouseewendy** [ko1/pretty_backtrace#7]
I've found the example can only work with script running, so I wonder will it support irb?

BTW, this gem really brings the conveniency, thanks :smile:

**Comment by ko1** [comment]
You are true. irb modify backtrace by their own knowledge and pretty_backtrace conflicts with this modification. I have no idea how to solve this issue.

Maybe this is essential problem that it conflicts with other libraries using Exception#set_backtrace method.
